poleno/attachments/admin.py

```diff
@@ -14,6 +16,11 @@ class DownloadAdminMixin(admin.ModelAdmin):
 
     def download_view(self, request, obj_pk):
         obj = self.model.objects.get_or_404(pk=obj_pk)
+        try:
+            if obj.content is None:
+                raise Http404()
```

Comment:
@martinmacko47 Mame to tak aj na inych miestach v kode. Preco niekde pouzivame `raise Http404()` so zatvorkami, a niekde `raise PermissionDenied` bez zatvoriek. Funkcnost je rovnaka. Je to vtedy ked chceme do Exception pridat aj nejaku spravu?

Reply:
@viliambalaz Dobra otazka. Ak funguju obe moznosti, aj so zatvorkami, aj bez zatvoriek. Tak to mozeme niekedy upratat. Asi by som pouzival vzdy moznost bez zatvoriek. Okrem pripadov, ze chceme dat do exception nejaky parameter.

Reply:
:ok: Ulozil som si tuto diskusiu, niekedy neskor pozriem celu codebase a upracem to.